### PR TITLE
Upgrade to support 1.16.4

### PR DESCRIPTION
**Pull request by ifvictr**

### CODE REVIEW

**Comment by eankeen** [comment]
looks good!

File: src/main/java/com/hackclub/hccore/listeners/AdvancementListener.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import org.bukkit.Statistic;
 import org.bukkit.advancement.Advancement;
 import org.bukkit.advancement.AdvancementProgress;
-import org.bukkit.craftbukkit.v1_16_R1.advancement.CraftAdvancement;
+import org.bukkit.craftbukkit.v1_16_R3.advancement.CraftAdvancement;
```
Comment (eankeen) [inline src/main/java/com/hackclub/hccore/listeners/AdvancementListener.java:10]: Interesting this isn't `v1_16_R4`?

Comment (ifvictr) [reply src/main/java/com/hackclub/hccore/listeners/AdvancementListener.java:10]: Nope, this is the NMS version. See https://www.spigotmc.org/wiki/spigot-nms-and-minecraft-versions-1-16/

**Comment by zanedb** [comment]
Just tested, looks good to me! Merge?

**Review by nice6599 — APPROVED** [review]
Double Checked. It works

**Pull request closed without merging**